PythonVisualizations/QuadTree.py

```diff
@@ -0,0 +1,227 @@
+import random
+from tkinter import *
+from VisualizationApp import *
+
+
+class Node(object):
+    def __init__(self, x, y, d):
+        self.x = x
+        self.y = y
+        self.data = [d] # might be more than one data at this coord 
+        self.dataObjects = [] #keeps track of text objects
+        self.NE = self.SE = self.SW = self.NW = None
+        self.countSpaces = 0
+        #used for drawing intersecting lines
+        self.horizontal_line = None
+        self.vertical_line = None
+
+class PointQuadtree(VisualizationApp):
+    MAX_ARG_WIDTH = 3
+    LINE_COLOR = 'SteelBlue4'
+    CIRCLE_DIMEN = 8
+     
+    def __init__(self, maxArgWidth = MAX_ARG_WIDTH, title="Point Quad Tree", **kwargs):
+        super().__init__(title=title, maxArgWidth = maxArgWidth, **kwargs)
+        self.__root = None
+        self.buttons = self.makeButtons()
+        self.points = []
+        self.nodes = []
+        self.lines = []
+        self.COUNTER = 1
```

Comment:
Shouldn't the `__init__()` and `new()` methods set the `COUNTER` to the same value?